Introduce Code of Conduct (#2)
apollographql/.github: CODE_OF_CONDUCT.md

## Apollo Code of Conduct

If you see a Code of Conduct violation, follow these steps:

- Let the person know that what they did is not appropriate and ask them to stop and/or edit their message(s) if online.
- That person should immediately stop the behavior and correct the issue.

> - If this doesn’t happen, or if you’re uncomfortable speaking up, contact organizers by emailing [codeofconduct@graphql.com](mailto:codeofconduct@graphql.com).

Reviewer: yes, that is the correct email address!

- As soon as available, an organizer will respond, identify themselves, and take further action (see below), starting with a warning, then temporary deactivation, then long-term deactivation.
- When reporting, please include any relevant details, links, screenshots, context, or other information that may be used to better understand and resolve the situation.
- The Admin team will prioritize the well-being and comfort of those affected negatively by the violation over the comfort of the violator. This may include the violator being removed from online or in-person spaces until those affected say they would feel safe with the person returning.

### Our Goal

This community is dedicated to providing a harassment-free experience for everyone. We do not tolerate harassment of participants in any form.

### Applicability and Scope

This code of conduct applies to all of this community's spaces, including public channels, private channels and direct messages, both online and off. Anyone who violates this code of conduct may be sanctioned or expelled from these spaces at the discretion of the administrators.

> ### Toward a Welcoming and Safe Environment

Reviewer: Open to changing this

> ### Toward a Welcoming and Safe Environment

Author: @kkemple I guess my only suggestion is to remove `Toward` or replace it with oneof the many suggestions below.  (Maybe it's just a usage of the word _toward_ I'm unfamiliar with — it does seem to be that we are trying to _point in the direction of_ this, so not wrong, I guess? idk.)

```suggestion
### A Welcoming and Safe Environment
```

```suggestion
### Cultivate a Welcoming and Safe Environment
```

```suggestion
### Encourage a Welcoming and Safe Environment
```

```suggestion
### Maintain a Welcoming and Safe Environment
```

### Enforcement Process

Every code of conduct violation report will be treated with seriousness and care. If a member's immediate safety and security is threatened, an individual admin may take any action that they deem appropriate, up to and including temporarily banning the offender from the community. In less urgent situations, at least two admins will discuss the offense and mutually arrive at a suitable response, which will be shared with the offender privately. Whatever the resolution that they decide upon, the decision of the admins involved in a violation case will be considered final.

We will respect confidentiality requests for the purpose of protecting victims of abuse. At our discretion, we may publicly name a person about whom we've received harassment complaints, or privately warn third parties about them, if we believe that doing so will increase the safety of our members or the general public. We will not name harassment victims without their affirmative consent.

> GraphQL Summit prioritizes marginalized people’s safety over privileged people’s comfort. The administrators will not act on complaints regarding:

Author: ```suggestion
Apollo prioritizes marginalized people’s safety over privileged people’s comfort. The administrators will not act on complaints regarding:
```

> - “Reverse”-isms, including “reverse racism,” “reverse sexism,” and “cisphobia”
> - Reasonable communication of boundaries, such as “leave me alone,” “go away,” or “I’m not discussing this with you.”
> - Communicating in a “tone” you don’t find congenial

Author: Fix magic smart quotes (which are inconsistent with this docs usage otherwise).

```suggestion
- “Reverse”-isms, including "reverse racism," "reverse sexism," and "cisphobia"
- Reasonable communication of boundaries, such as "leave me alone," "go away," or "I’m not discussing this with you."
- Communicating in a "tone" you don’t find congenial
```
